src/main/java/org/entur/gbfs/GbfsLoader.java

```diff
@@ -94,6 +99,21 @@ public GbfsLoader(String url, Map<String, String> httpHeaders, String languageCo
      * Checks if any of the feeds should be updated base on the TTL and fetches. Returns true, if any feeds were updated.
      */
     public boolean update() {
+        try {
+            boolean canUpdate = updateLock.tryLock(5, TimeUnit.SECONDS);
```

Comment:
Should the timeout be configurable, or should we even set it to zero, to just say that a concurrent update didn’t provide any new data.

Reply:
Yes I'm inclined to just use tryLock without any timeout, which will immediately return false if another thread is holding the lock.